src/cfn/terminalStackStates.ts

```diff
@@ -0,0 +1,28 @@
+// http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-listing-event-history.html
```

Comment:
I think the other status lists could be colocated together, in something like a `statuses` module.